src/Swarm/Game/Step.hs

```diff
@@ -548,12 +548,14 @@ evalConst c vs s k = do
 seedProgram :: Integer -> Integer -> Text -> ProcessedTerm
 seedProgram minTime randTime thing =
   [tmQ|
-    r <- random (1 + $int:randTime);
-    wait (r + $int:minTime);
-    appear "|";
-    r <- random (1 + $int:randTime);
-    wait (r + $int:minTime);
-    place $str:thing;
+    try {
+      r <- random (1 + $int:randTime);
+      wait (r + $int:minTime);
+      appear "|";
+      r <- random (1 + $int:randTime);
+      wait (r + $int:minTime);
+      place $str:thing;
+    } {};
     selfdestruct
```

Comment:
Your main branch seems to be behind? Please sync your main with byorgey/main. :slightly_frowning_face: 

I usually branch from byorgey/main itself, but you should have a button in your repo that says **"Fetch upstrem"**.

That might do the trick, if not, rebase the last commit on your main :slightly_smiling_face:

Reply:
Yes, but you did not update *your* main branch and then you branched from it :man_shrugging: 

It probably would not cause any issue when merging, but I would prefer keeping PRs clean of such noise. :slightly_smiling_face:

Reply:
See below, the blue branch is this one, white is your main and orange is byorgey/main:

![Screenshot from 2022-02-01 20-47-34](https://user-images.githubusercontent.com/44544735/152040770-18aaa0f3-54fc-433a-ad15-cd39189ca4c6.png)

What I would like is, if the white was same as blue (that is your fech upstream into your main) and this branch would go from that updated main instead of the branch you used in #298 (I guess use git rebase).

Sorry to pester you about this, if you don't know how to use rebase, don't worry about it and the merge will hopefully work.
But then please switch back to branching from main. :smile: